src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs

```diff
@@ -406,15 +406,15 @@ public void SetSelectedPathOrNavigate(PathNavigationEventArgs e)
 					}
 				}
 			}
-			if (PathNormalization.NormalizePath(ParentShellPageInstance.FilesystemViewModel.WorkingDirectory) !=
-				PathNormalization.NormalizePath(e.ItemPath))
-			{
+
+			if (ParentShellPageInstance is null)
+				return;
```

Comment:
Should we also call `DismissOtherBlades`?

Reply:
I don't think so.